android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleTagManagerBridge.java

```diff
@@ -39,6 +50,13 @@ public String getName() {
         return "GoogleTagManagerBridge";
     }
 
+    @Override
+    public Map<String, Object> getConstants() {
```

Comment:
I'd rather not export constants via bridge. The only consumer here is this library, and exporting through bridge means we have to setup on main thread on iOS.

Since we already know the constants, I'd rather have them predefined in JS & native code. (Yes, this means duplicate/redundant code, but we avoid main thread issue). For the next major version of the library I am removing all constant export from bridge.

Reply:
Fair enough. 👍🏻